server/service/service.go

```diff
@@ -34,6 +33,13 @@ func New(conf *config.Config) (*Service, error) {
 		return nil, fmt.Errorf("failed to create PanaceaGRPCClient: %w", err)
 	}
 
+	registeredDataValidator, err := panaceaClient.GetRegisteredDataValidator(validatorAccount.GetAddress())
+	if err != nil {
+		return nil, err
+	} else if registeredDataValidator == nil {
+		return nil, fmt.Errorf("this data validator is not registered in Panacea yet")
+	}
```

Comment:
My guess is that there is no case where `registeredDataValidator` is empty without an error.
There is a way to delete this part.
```go
else if registeredDataValidator == nil {
	return nil, fmt.Errorf("this data validator is not registered in Panacea yet")
}
```

Or, for a more accurate response, check for duplicate errors and respond to this message.(`this data validator is not registered in Panacea yet`)

Reply:
I agree with your comment. 
Plus, Panacea will return error message but it is not suitable for data validator.
I'll change it.

Reply:
```go
_, err = panaceaClient.GetRegisteredDataValidator(validatorAccount.GetAddress())
if err != nil {
      if strings.HasSuffix(err.Error(), datapooltypes.ErrDataValidatorNotFound.Error()) {
           return nil, fmt.Errorf("this data validator is not registered in Panacea yet")
      }
      return nil, err
}
```
How about change like this?

Reply:
Yes, It seems good.